Editor/Scripts/UserSettings/DCCToolsSettingsTab.cs

```diff
@@ -124,20 +124,48 @@ internal class DCCToolsSettingsTab : IMeshSyncSettingsTab{
 //----------------------------------------------------------------------------------------------------------------------        
 
     #region Button callbacks
+
+    private const string addDCCToolLabel = "Add DCC Tool";
+    
+    string GetFilePath() {
```

Comment:
Can you please change the name to indicate that this opens a FilePanel ?

Reply:
done